Cocktail/Cocktail/ViewControllers/CocktailDetailViewController.swift

```diff
@@ -193,6 +210,14 @@ class CocktailDetailViewController: UIViewController {
         recipeLabel.text = data.recipe.localized
         myTipLabel.text = data.mytip.localized
         ingredientsLabel.text = data.ingredients.map {$0.rawValue.localized}.joined(separator: ", ")
+        cocktailData = data
+        
+        if data.wishList == true {
+            likeButton.setImage(UIImage(systemName: "heart.fill"), for: .normal)
+        } else {
+            likeButton.setImage(UIImage(systemName: "heart"), for: .normal)
+        }
```

Comment:
```suggestion
        likeButton.setImage(UIImage(systemName: data.wishList == true ? "heart.fill" : "heart"), for: .normal)
```
이 부분도 역시 삼항조건으로 처리할 수 있을 듯